flink/flink-1.13/src/main/java/io/tidb/bigdata/flink/telemetry/AsyncTelemetry.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 TiDB Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.tidb.bigdata.flink.telemetry;
+
+import io.tidb.bigdata.telemetry.Telemetry;
+import io.tidb.bigdata.telemetry.Telemetry.ReportState;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AsyncTelemetry {
+  private static final Logger LOG = LoggerFactory.getLogger(AsyncTelemetry.class);
+
+  private Map<String, String> properties;
+
+  public AsyncTelemetry(Map<String, String> properties) {
+    this.properties = properties;
+  }
+
+  public void report() {
+    try {
+      if (FlinkTeleMsg.getInstance(properties).shouldSendMsg()) CompletableFuture.runAsync(task);
```

Comment:
Can you introduce the benefits of  using`CompletableFuture.runAsync`

Reply:
It's easy to use. Here,  the benefit of  CompletableFuture is that we needn't create a thread pool or build a new thread and run. You can find more information about CompletableFuture in this [link](https://mincong.io/2020/06/26/completable-future/)